app/views/shared/_modal_layout.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="usa-modal-wrapper is-visible">
   <div class="usa-modal-overlay">
     <%= tag.div(
-          role: 'dialog',
+          role: 'alertdialog',
```

Comment:
I am a little concerned about this because this layout is used on all the modals in the app. Is there a way we can use logic to determine if the role should be `alertdialog` vs `dialog`?

Reply:
Hmm, ok, I added locals parameter to the render and updated the three places where it is used.